include/solvers/postprocessing_cfd.h

```diff
@@ -57,13 +61,16 @@
  * @param fem_parameters The fem_parameters of the simulation
  *
  * @param mpi_communicator The mpi communicator. It is used to reduce the CFL calculation.
+ *
+ * @param simplex_enabled Is the simplex mode enabled
  */
 template <int dim, typename VectorType>
 double
 calculate_CFL(const DoFHandler<dim> &dof_handler,
               const VectorType &     evaluation_point,
               const double           time_step,
-              const MPI_Comm &       mpi_communicator);
+              const MPI_Comm &       mpi_communicator,
+              const bool             simplex_enabled);
```

Comment:
This simplex parameter will need to be remove. It should be straightforward though.